src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/aos/AcknowledgementOfClaimGenerator.java

```diff
@@ -37,9 +37,9 @@
     private final RepresentativeService representativeService;
 
     public CaseDocument generate(CaseData caseData, String authorisation) {
-        AcknowledgementOfClaimForm templateData = getTemplateData(caseData);
+        AcknowledgementOfClaimForm templateData = getTemplateDataForAcknowldgeClaim(caseData);
         DocmosisTemplates docmosisTemplate =
-            getMultiPartyScenario(caseData) == ONE_V_TWO_ONE_LEGAL_REP ? N9_MULTIPARTY_SAME_SOL : N9;
+            getMultiPartyScenario(caseData) == ONE_V_TWO_ONE_LEGAL_REP ? N9_MULTIPARTY_SAME_SOL : N11;
```

Comment:
Please consider using equals method instead of equality operator. same for the usage in following lines as well.